bin/terminus.php

```diff
@@ -38,6 +41,13 @@
     ->withArgument('fileCache');
 $container->inflector(SessionAwareInterface::class)
     ->invokeMethod('setSession', ['session']);
+$container->share('sites', Sites::class);
+$container->inflector(SiteAwareInterface::class)
+    ->invokeMethod('setSites', ['sites']);
+$container->share('user', User::class);
```

Comment:
Does it aid us to have a UserAwareInterface here? A User object could be used to represent any user, not just the current one (although we don't currently use it that way). I think it makes the logic clearer that we have to retrieve the user from the session thereby marking the user being operated on in the command explicitly the one who is logged in.